editor/src/components/canvas/canvas-utils.ts

```diff
@@ -169,6 +169,7 @@ import { fastForEach } from '../../core/shared/utils'
 import { UiJsxCanvasContextData } from './ui-jsx-canvas'
 import { addFileToProjectContents, contentsToTree } from '../assets'
 import { openFileTab } from '../editor/store/editor-tabs'
+import {emptyComments} from "../../core/workers/parser-printer/parser-printer-comments";
```

Comment:
I have a horrible feeling we're going to see all of these import statements change in a future PR as a result of something prettier or eslint related, because of the missing space inside the braces, and the quotation marks :grimacing:

Reply:
oh I didn't notice these! didn't I fix the tests to break if something is not formatted according to prettier? or did we just talk about this?

in any case, you can run `prettier` from the command line and tell it to go over a list of files, or every file